### ISSUE TITLE
ids need to be unique

### ISSUE THREAD

**Issue by farishkash** [galacticdragon/prj-rev-bwfs-dasmoto#1]
Example

<div id="item"> is used multiple times.

This can cause issues in the future when it comes to anchor links and jquery selectors.

You can use classes for this purpose instead.

<div class="item">

**Comment by galacticdragon** [comment]
so basically each element category can have the same class, but each instance must have a different id?

**Comment by farishkash** [comment]
Yes you can repeat the classes infinitely .  As a general rule I only use ids for in page linking or for specific jquery targeting.
